Shared/Flows/Photo Capture/ExpressionPhotoCaptureViewController.swift

```diff
@@ -32,7 +32,12 @@ class ExpressionPhotoCaptureViewController: ViewController {
 
         self.tapView.didSelect { [unowned self] in
             if self.faceCaptureVC.isSessionRunning {
-                self.faceCaptureVC.capturePhoto()
+//                self.faceCaptureVC.capturePhoto()
+
+                self.faceCaptureVC.startVideoCapture()
+                delay(2) {
+                    self.faceCaptureVC.finishVideoCapture()
+                }
```

Comment:
For testing, I'm just using the existing tap and stopping video capture after 2 seconds.